docs/en/latest/plugins/api-breaker.md

```diff
@@ -90,7 +95,7 @@ curl "http://127.0.0.1:9080/apisix/admin/routes/1" -H 'X-API-KEY: edd1c9f034335f
 Then. Like the configuration above, if your upstream service returns 500. 3 times in a row. The client will receive a 502 (break_response_code) response.
 
 ```shell
-$ curl -i -X POST "http://127.0.0.1:9080/get"
+$ curl -i -X POST "http://127.0.0.1:9080/hello"
```

Comment:
A route is not configured api-breaker and which have a unavailable upstream, APISIX will return 502 error, so your example is not enough to say it's due to the 502 error returned by the api-breaker plugin.

Reply:
Yes. Maybe we can change the status code.

Most of the documentation is using the 1980 port as a fake upstream. So we can assume people can get used to it.

We can't use 9080 port as it may cause a loop.
`httpbin.org` is another option, but it can't be accessed in the China mainland, and it only supports `http`.